e2e-testing/README.md

```diff
@@ -16,3 +16,33 @@ In another terminal, run
 ```
```

Comment:
nit: Did you add a github action to run these e2e tests or a github action to at least check that these files pass build and lint tests?

Reply:
will do this separately. Want to make sure it's not flaky in github actions before submitting. created ticket here: https://linear.app/dydx/issue/IND-550/github-actions-for-buildingrunning-e2e-test